tools/project_manager/handle_bad_page.php

```diff
@@ -27,6 +27,14 @@
     die("You are not authorized to manage this project.");
 }
 
+$project = new Project($projectid);
+
+// prevent changes to the project table if it isn't UTF-8
+if(!$project->is_utf8)
+{
+    die(_("Project table is not UTF-8."));
+}
```

Comment:
I'm assuming that the way this message is provided, there's no way to do `class='error'`? Is the same true of including a back-link?